src/wfs/src/main/java/org/geoserver/wfs/DeleteElementHandler.java

```diff
@@ -46,10 +46,13 @@ public class DeleteElementHandler extends AbstractTransactionElementHandler {
     /** logger */
     static Logger LOGGER = org.geotools.util.logging.Logging.getLogger("org.geoserver.wfs");
 
-    FilterFactory factory = CommonFactoryFinder.getFilterFactory(null);
+    FilterFactory filterFactory;
+    private final WFSTransactionExceptionFactory exceptionFactory;
```

Comment:
Seems to be in most transaction handlers, in that case, why not place it in the abstract base class instead?

Reply:
Extracted up to abstract base.